src/Descriptor.hpp

```diff
@@ -53,13 +53,13 @@ class Descriptor {
     uint16_t port_{};
     bool processing_command_{};
     DescriptorState state_{DescriptorState::GetName};
+    CHAR_DATA *character_{};
```

Comment:
ok they're private now so that'll  force everyone to use the new accessors, cool